manuscript/04-context-mapping-fuer-strategic-design.md

```diff
@@ -308,7 +308,7 @@ immer mehr zu beachten, wenn Änderungen vorgenommen werden.
 Daher:
 
 **Definiere ein Protokoll, das den Zugriff auf dein Subsystem als eine
-Menge von Diensten ermöglicht. Öffne das Protokoll, so dass
+Menge von [Services](#service) ermöglicht. Öffne das Protokoll, so dass
```

Comment:
Mein Verständnis ist hier anders. Das Service-Pattern will meiner Meinung nach Aussagen, dass nicht alle Fachlogik natürlich auf eine Entität passt, und sie in solchen Fällen dann auf einen Service gelegt werden soll. Service im Rahmen vom "Open Host Service" sehe ich eher als abstrakter / generischer, einfach als Schnittstelle. Ob dadurch dann Funktionalität angestossen wird, die auf einem Service (im ersten Sinn) oder auf einer Entität liegt, ist dabei meiner Ansicht nach irrelevant.

Natürlich hast Du mit den "Kapitälchen" ein Argument, aber vielleicht ist das ja auch einfach ein Versehen des Editors des blauen Buches (Search - Replace)? Im Zweifel würde ich von der Reference des Musters absehen, oder das versuchen mit Eric zu klären. Die Referenz kann auch irreführend sein (eben z.B. was mache ich, wenn ich Funktionalität meines Domänenmodells zugänglich machen will, die nicht auf einem Service liegt? Muss ich dann nur dafür einen Service einführen? Fragen dieser Art ...)